src/tasks/AndroidAppBuilder/Templates/MonoRunner.java

```diff
@@ -68,24 +69,24 @@ public void onCreate(Bundle arguments) {
         start();
     }
 
-    private static String getDocsDir(Context ctx) {
-        File docsPath  = ctx.getExternalFilesDir(Environment.DIRECTORY_DOCUMENTS);
-        if (docsPath == null) {
-            docsPath = ctx.getCacheDir();
-        }
-        return docsPath.getAbsolutePath();
-    }
-
     public static int initialize(String entryPointLibName, String[] args, Context context) {
         String filesDir = context.getFilesDir().getAbsolutePath();
         String cacheDir = context.getCacheDir().getAbsolutePath();
-        String docsDir = getDocsDir(context);
+
+        File docsPath = context.getExternalFilesDir(Environment.DIRECTORY_DOCUMENTS);
```

Comment:
What's the value of `DIRECTORY_DOCUMENTS`?

Reply:
I am asking because before we were setting env var `DOCSDIR` and I don't understand whether this is the same?

Reply:
You mean the enum value? It's the string `Documents`.

Reply:
Ah, so it's something from Android